$MirOS$

	① Not needed, CPPFLAGS are already right in MirPorts.
	② Install MLINKS

--- vm/Makefile.inc.orig	Sun May 11 23:15:25 2008
+++ vm/Makefile.inc	Sun Oct  5 15:17:30 2008
@@ -34,19 +34,6 @@ CPPFLAGS+=	-DPREFIX=\"${PREFIX:Q}\"
 # possible values: gtk1 gtk2 cli
 USE_TOOLKIT?=	gtk1
 
-.ifndef JPEG_PREFIX
-.  if exists(/usr/include/jpeglib.h)
-JPEG_PREFIX=	/usr
-.  elif exists(/usr/mpkg/include/jpeglib.h)
-JPEG_PREFIX=	/usr/mpkg
-.  elif exists(/sw/include/jpeglib.h)
-JPEG_PREFIX=	/sw
-.  endif
-.endif
-JPEG_PREFIX?=	${PREFIX}
-
-CPPFLAGS+=	-I${JPEG_PREFIX}/include
-LDFLAGS+=	-L${JPEG_PREFIX}/lib
 LDADD+=		-ljpeg
 
 .if ${USE_TOOLKIT} == "gtk1"
@@ -87,6 +74,16 @@ classes.o: ../devel/ewe.ewe
 .if exists(${.SYSMK}/libmirmake.a)
 LDADD+=		-L${.SYSMK:Q} -lmirmake
 CPPFLAGS+=	-D_MIRMAKE_DEFNS -include ${.SYSMK:Q}/mirmake.h
+.endif
+
+.if ${NOMAN:L} == "no"
+MLINKS+=	ewe.1 ewene.1
+MLINKS+=	ewe.1 ewecl.1
+MLINKS+=	ewe.1 eweclne.1
+MLINKS+=	ewe.1 eweg2.1
+MLINKS+=	ewe.1 eweg2ne.1
+MLINKS+=	ewe.1 ewedev.1
+MLINKS+=	ewe.1 ewemp.1
 .endif
 
 .endif
